classes/Mail.php

```diff
@@ -219,11 +219,11 @@ public static function Send(
                 }
 
                 if (is_array($toName) && $toName && is_array($toName) && Validate::isGenericName($toName[$key])) {
-                    $toName = $toName[$key];
+                    $addrName = $toName[$key];
                 }
 
-                $toName = (($toName == null || $toName == $addr) ? '' : self::mimeEncode($toName));
-                $message->addTo($addr, $toName);
+                $addrName = (($toName == null || $toName == $addr) ? '' : self::mimeEncode($toName));
```

Comment:
How come you reassign `$addrName` here? The above if doesn't do anything if the variable isn't used between 2 assignments?